builtin/providers/circonus/resource_circonus_check_http_test.go

```diff
@@ -0,0 +1,180 @@
+package circonus
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccCirconusCheckHTTP_basic(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckDestroyCirconusCheckBundle,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCirconusCheckHTTPConfig,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "active", "true"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "collector.#", "1"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "collector.2388330941.id", "/broker/1"),
+					resource.TestCheckResourceAttr("circonus_check.jezebel", "http.#", "1"),
+					// resource.TestCheckResourceAttr("circonus_check.jezebel", "http.4213422905.auth_method", ""),
```

Comment:
Everything is testable.  :)  These are commented out because the ROI on testing every individual return parameter is pretty low.

In this case, we'd need to put together a valid end-to-end test and viable endpoint.  There are a number of permutations and combinations for how these values could be tested, but we don't have an endpoint that we could easily rig up.